common/numpy_fast.py

```diff
@@ -1,2 +0,0 @@
-def clip(x, lo, hi):
```

Comment:
numpy_fast exists for performance. For small arrays (<10000), it is faster than numpy.